### PR TITLE
Bakers count

### PR DESCRIPTION
**Pull request by dvanderbeek**
[Notion Task](https://www.notion.so/figmentnetworks/Counter-Cache-bakers-141ae53d2eff425893e881d8ba8903ae)

Adds a counter_cache column to store the number of bakers associated with a Tezos Chain and an API endpoint to retrieve this data.

### CODE REVIEW

File: app/controllers/tezos/bakers_controller.rb [hunk 1]
```diff
@@ -8,6 +8,12 @@ def index
     @tezos_bakers = @tezos_bakers.where("LOWER(id) = ? OR name ILIKE ?", params[:query].downcase, "%#{params[:query]}%") if params[:query]
   end
 
+  # GET /tezos/bakers/count.json
+  def count
+    @chain = Tezos::Chain.primary
```
Comment (avanderbeek) [inline app/controllers/tezos/bakers_controller.rb:13]: Why set @chain and then @count?  Don't seem to use @chain.

**Comment by avanderbeek** [comment]
Other than minor comment aboce this looks good.

File: app/controllers/tezos/bakers_controller.rb (hunk 1 above)
Comment (dvanderbeek) [reply app/controllers/tezos/bakers_controller.rb:13]: Yeah I meant to change that. Good catch

**Comment by dvanderbeek** [comment]
@pawelnguyen I added a test to the corresponding PR on Hubble. For this, I'll need to get RSpec set up, but I should be able to do that pretty quickly and add some coverage here.

**Comment by dvanderbeek** [comment]
@pawelnguyen I set up RSpec and FactoryBot for the Tezos indexer and added a couple tests to cover this feature

**Pull request merged**